app/javascript/controllers/form_controller.js

```diff
@@ -0,0 +1,32 @@
+import { Controller } from "@hotwired/stimulus";
+
+export default class extends Controller {
```

Comment:
**nitpick:** Can you rename this file?  I just went to it and had no idea what it was for (until I read the code).

Reply:
I named it the same as the RoR controller. Please see 6cef0e01dbb9ee435d8727f934c83c6cc8d92bc1. If that's no good, could you please offer a suggestion.